nemo_curator/utils/distributed_utils.py

```diff
@@ -216,7 +218,7 @@ def read_single_partition(
             " file formats.."
         )
 
-    if filetype == "jsonl":
+    if filetype in ["jsonl", "json"]:
         read_kwargs = {"lines": True}
```

Comment:
This `{"lines": True}` won't work if `filetype== "json"`, right? Probably best to change this to `{"lines": filetype == "jsonl" }`

Reply:
https://github.com/NVIDIA/NeMo-Curator/pull/130/commits/a304629f572c0bb202bd14efa5843b18bb7ca388